docker/greenmask/Dockerfile

```diff
@@ -38,4 +38,7 @@ RUN mkdir ~/.bash_completions \
     && echo 'source /etc/bash_completion' >> /root/.bashrc \
     && echo 'source /root/.bash_completions/greenmask.bash' >> /root/.bashrc
 
-ENTRYPOINT ["/bin/bash"]
+RUN groupadd -r greenmask && useradd -r -g greenmask greenmask
+USER greenmask
+
+ENTRYPOINT ["greenmask"]
```

Comment:
There is the playground as well, ensure that it can be run correctly with `docker-compose run greenmask`. I suspect you should override entrypoint in docker-compose.

You can check it without docker hub publishing via:

`docker-compose run greenmask-from-source`

Reply:
But I've just checked the implementation. It should work fine there is already an overridden entrypoint. But check it please

Reply:
@wwoytenko after running the test I figured that I missed the bash completion section on the Dockerfile that writes to the root home folder. The playground script also writes to it, so I also had to change it